app/grandchallenge/algorithms/templates/algorithms/data_tables/job_list.html

```diff
@@ -33,6 +33,7 @@
         {% firstof job.output|user_error job.get_status_display %}
    {% endif %}"
    data-pk="{{ job.pk }}"
+   data-rendered-output="{{ job.rendered_result_text }}"
```

Comment:
This doesn't work when the rendered_result_text contains quotes, and I'm not sure what would be the right way to handle it. I think I can't set it as a property this way?

Reply:
I think that `job.rendered_result_text|escape`  might suffice here to fix that issue.